config/config.go

```diff
@@ -118,6 +118,9 @@ type Config struct {
 	// TODO: We actually only support mode 2, which keeps the original case, but the comparison is case-insensitive.
 	LowerCaseTableNames        int                `toml:"lower-case-table-names" json:"lower-case-table-names"`
 	ServerVersion              string             `toml:"server-version" json:"server-version"`
+	VersionComment             string             `toml:"version-comment" json:"version-comment"`
+	TiDBEdition                string             `toml:"tidb-edition" json:"tidb-edition"`
+	TiDBReleaseVersion         string             `toml:"tidb-release-version" json:"tidb-release-version"`
```

Comment:
Usually when we update `config.go`, [`config.toml.example`](https://github.com/pingcap/tidb/blob/master/config/config.toml.example) is updated at the same time.

But this case is a little special since it is meaningless for most of the users. IMHO it doesn't have to be listed in example file.